README.md

```diff
@@ -15,61 +21,307 @@ Our other repositories are:
 
-1. Run the project setup (If there are errors, check for Python versions >=3.9,<3.12. Ensure that you have the required database set up and migrations)
+**Note:** You will also need Xcode or the Xcode Command Line Tools installed. The
+quickest way to do this is is by installing the command line tools in the shell:
 
-   `make bootstrap`
+```sh
+xcode-select –-install
+```
+
+#### Homebrew Setup
+
+If this is your first time installing Homebrew on your machine, you may need to
+add its binaries to your system's `$PATH` environment variable so that you can
+use the `brew` command.  Try running `brew help` to see if Homebrew is
+recognized and runs properly.  If that fails, then you'll need to add a
+configuration line to wherever your `$PATH` environment variable is set.
+
+Your system `$PATH` environment variable is likely set in one of these
+locations:
+
+For BASH shells:
+- `~/.bashrc`
+- `~/.bash_profile`
+- `~/.profile`
+
+For ZSH shells:
+- `~/.zshrc`
+- `~/.zprofile`
+
+There may be different files that you need to modify for other shell
+environments.
+
+Which file you need to modify depends on whether or not you are running an
+interactive shell or a login shell
+(see [this Stack Overflow post](https://stackoverflow.com/questions/18186929/what-are-the-differences-between-a-login-shell-and-interactive-shell)
+for an explanation of the differences).  If you're still not sure, please ask
+the team for help!
+
+Once you determine which file you'll need to modify, add these lines before any
+lines that add or modify the `$PATH` environment variable; near or at the top
+of the file is appropriate:
+
+```sh
+# Homebrew setup
+eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
+This will make sure Homebrew gets setup correctly. Once you make these changes,
+either start a new shell session or source the file
+(`source ~/.FILE-YOU-MODIFIED`) you modified to have your system recognize the
+changes.
 
-1. Run the web server and background workers
+Verify that Homebrew is now working by trying to run `brew help` again.
 
-   `make run-procfile`
+### System-Level Package Installation
 
-- Or run them individually:
+There are several packages you will need to install for your system in order to
+get the app running (and these are good to have in general for any software
+development).
 
-- Run Flask (web server)
+Start off with these packages since they're quick and don't require additional
+configuration after installation to get working out of the box:
 
-  `make run-flask`
+- [jq](https://stedolan.github.io/jq/) - for working with JSON in the command
+  line
+- [git](https://git-scm.com/) - for version control management
+- [tfenv](https://github.com/tfutils/tfenv) - for managing
+  [Terraform](https://www.terraform.io/) installations
+- [cf-cli@8](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) - for
+  working with a Cloud Foundry platform (e.g., cloud.gov)
+- [redis](https://redis.io/) - required as the backend for the API's
```

Comment:
i like how to added what they're used for. this is helpful to reference

Reply:
Thank you! :-)